System.Containers.Tasks/build/System.Containers.Tasks.targets

```diff
@@ -3,43 +3,68 @@
         <TargetFrameworkVersion>6.0</TargetFrameworkVersion>
     </PropertyGroup>
 
-    <!-- Required targets dependencies -->
-    <PropertyGroup>
-        <PublishContainerDependsOn>
-            Build;
-            ComputeContainerConfig
-        </PublishContainerDependsOn>
-    </PropertyGroup>
-
-    <Target Name="ComputeContainerConfig" AfterTargets="Build;_CopyResolvedFilesToPublishPreserveNewest">
-        <!-- Find everything in the publish directory -->
-        <ItemGroup>
-            <_BuildOutputsNonRelative Include="$(MSBuildProjectDirectory)\$(PublishDir)**\*.*"/>
-            <BuildOutputs Include="%(_BuildOutputsNonRelative.FullPath)"/>
-        </ItemGroup>
+    <Target Name="ComputeContainerConfig">
+        <!-- Compute private defaults -->
+        <PropertyGroup>
+            <_ContainerBaseRegistry Condition="'$(_ContainerBaseRegistry)' == ''">https://mcr.microsoft.com</_ContainerBaseRegistry>
+            <_ContainerBaseImageName Condition="'$(_ContainerBaseImageName)' == '' and @(ProjectCapability->AnyHaveMetadataValue('Identity', 'AspNetCore'))">dotnet/aspnet</_ContainerBaseImageName>
+            <_ContainerBaseImageName Condition="'$(_ContainerBaseImageName)' == ''">dotnet/runtime</_ContainerBaseImageName>
+            <_ContainerBaseImageTag Condition="'$(_ContainerBaseImageTag)' == ''">$(_TargetFrameworkVersionWithoutV)</_ContainerBaseImageTag>
```

Comment:
Do we need the conditions if these are just private defaults? Right now it's nontrivial to overwrite just one of these things, which seems ok--we can document only `ContainerBaseImage`

Reply:
The fact that they're private convinced me that this was indeed overkill. I'll condition them on `ContainerBaseImage` being empty, these privates aren't needed if the user has already set `ContainerBaseImage`